teammapper-frontend/.eslintrc.json

```diff
@@ -47,20 +57,21 @@
             "style": "camelCase",
             "type": "attribute"
           }
-        ],
-        "@typescript-eslint/member-delimiter-style": [
-          "warn",
-          {
-            "multiline": {
-                "delimiter": "semi",
-                "requireLast": true
-            },
-            "singleline": {
-                "delimiter": "semi",
-                "requireLast": false
-            }
-          }
         ]
+        // @TODO: Replace with ESLint Stylistic: https://typescript-eslint.io/blog/deprecating-formatting-rules/#upgrading-to-eslint-stylistic
```

Comment:
do you want to keep this open? than we should add an issue. Otherwise we can also add it here

Reply:
I already made an issue for it: #409

But I've removed the rule for now so this comment won't show up on main.